src/Components/LayoutType.elm

```diff
@@ -95,6 +95,8 @@ type alias ShowRightSidebarMenu msg =
     , localMsg : msg
     , audioMsg : msg
     , videoMsg : msg
+    , currentlyPlaying : Maybe ItemDetails
```

Comment:
Do you see a way to use `currentlyPlaying` from the `CurrentlyPlaying` state?

Reply:
Please correct me if I am wrong since, both CurrentlyPlaying and ShowRightSidebarMenu are getting currentlyPlaying value from the Shared.model. 
```
(Shared.elm)
  , currentlyPlaying =
            { currentlyPlaying = model.currentlyPlaying
            , progressSlider = Element.map toMsg (slider model.progressSlider)
            }
  , showRightSidebarMenu =
            { ....
            , currentlyPlaying = model.currentlyPlaying
            ....
            }
```
I don’t think we need to use currentlyPlaying from CurrentlyPlaying state

Reply:
We don't need to do that. But it's the right thing to do, we don't want fields that are the same thing mulitple times in our state, as they might drift from each other. So best practice would be to try to only have one field for every data point.

Unless we find out, that this is a different thing and somehow specific, but then our naming should communicate that too.